WordPress/Classes/Models/Blog.m

```diff
@@ -529,20 +524,11 @@ - (BOOL)accountIsDefaultAccount
     return [accountService isDefaultWordPressComAccount:self.account];
 }
 
-- (BOOL)jetpackAccountIsDefaultAccount
-{
-    AccountService *accountService = [[AccountService alloc] initWithManagedObjectContext:self.managedObjectContext];
-    return [accountService isDefaultWordPressComAccount:self.jetpackAccount];
-}
-
 - (nullable NSNumber *)siteID
```

Comment:
Since we're removing the declaration from the header can we nuke the method outright?  I don't think its been called anywhere else.